targets/csharp/source/PlayFabSDK/source/Uunit/PluginManagerTest.cs

```diff
@@ -113,6 +115,34 @@ public void PluginManagerMultiplePluginsPerContract(UUnitTestContext testContext
             testContext.True(object.ReferenceEquals(actualTransport2, expectedTransport2));
             testContext.EndTest(UUnitFinishState.PASSED, null);
         }
+
+        /// <summary>
+        /// Test that plugin manager can set and return the polly plugin.
+        /// </summary>
+        [UUnitTest]
+        public void PluginManager_PollyPlugin_Succesful(UUnitTestContext testContext)
```

Comment:
This is a good test, but needs to be refactored to use the PlayFab_Transport plugin contract. In theory, this should be mimicking what a game would do to change their transport plugin from the default to your polly plugin.